gascity/skills/adt-tools/SKILL.md

```
@@ -0,0 +1,63 @@
+
+# adt-tools
+
+This skill documents how Gas City agents use the `adt` CLI and `adt-mcp`
+API that are wired into the city as pack commands.
+
+## Setup
+
+Make sure the `adt-cli` rig points at the `abapify/adt-cli` checkout and
+the packages are built:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)